src/popup/pages/Send/index.js

```diff
@@ -147,8 +194,24 @@ class SendPage extends React.Component {
         toAddressTitle=getLanguage('toAddress')
         confirmTitle = getLanguage('sendDetail')
         confirmToAddressTitle = getLanguage('toAddress')
-
         isWithdraw = true
+
+        warnBeforeSending = async () => {
+          const ownAddresses = this.state.allAccounts.commonList
+            .filter(acc => {
+              return (
+                acc.type === ACCOUNT_TYPE.WALLET_INSIDE ||
+                acc.type === ACCOUNT_TYPE.WALLET_LEDGER ||
+                acc.type === ACCOUNT_TYPE.WALLET_OUTSIDE
+              )
+            })
+            .map(acc => acc.address)
+
+          if (!ownAddresses.includes(this.state.toAddress)) {
+            return getLanguage("confirmWithdrawingFromParatimeToForeignAccount", "Destination account is not in your wallet! Some automated systems, e.g., those used for tracking exchange deposits, may be unable to accept funds through ParaTime withdrawals. For better compatibility, cancel, withdraw into your own account, and transfer from there.")
```

Comment:
A note for later, if we ever re-enable displaying the oasis1... address for secp256k1 keys: this check will still prevent people from withdrawing into those addresses, but the message might become a little confusing. We could add an alternative message at that time.

Reply:
(and message is already slightly inaccurate if user has destination address imported in `watch mode`)